Docker/Compose/pgsql/env-idp.example

```diff
@@ -0,0 +1,47 @@
+ADMIN_API_VIRTUAL_NAME=adminapi
```

Comment:
Noticed that this file is named `env-idp.example` for pgsql, but for mssql it is `.env-idp.example`